### DISCUSSION TITLE
Changing the  "GiveWP" text in the stripe banking modal, to my own website name?

### DISCUSSION THREAD

**Discussion by demo7up** [impress-org/givewp#7615] [Developer Q&A]
I would like to change the "GiveWP" text In the stripe banking modal, to my own website name instead of the plugin name. It has to be piped in somewhere via a variable, function, or even if you have a hook / action to change it please guide me in that direction. Thanks
![Screenshot 2024-11-06 125803](https://github.com/user-attachments/assets/94bc11ef-b51a-46f7-ab66-dc5afd58adf5)

**Comment by demo7up** [comment]
Bump

**Comment by jonwaldstein** [comment]
Hey @demo7up 

I looked into this and had to contact Stripe.  Apparently, there is a setting in our GiveWP account specifically for what name to display for financial institutions. I went ahead and changed this to the connected account's name.  So in theory it should display your Stripe account name now! 😄 

![Screenshot 2024-11-19 at 4 42 16 PM](https://github.com/user-attachments/assets/04f4f034-269d-47f4-baea-f159380a9c02)

**Comment by demo7up** [comment]
Unfortunately its still not working for me, Under my own stripe account its showing as it should be if you see in the last screenshot it still shows as GiveWP on my live site. Ive also disconnected my stripe account and reconnected it to GiveWP as a payment method and its still showing the same.

![Screenshot 2024-11-20 220958](https://github.com/user-attachments/assets/30a0e602-2062-4bd0-83eb-71535c60751a)
![Screenshot 2024-11-20 220937](https://github.com/user-attachments/assets/92ab77dc-9c97-4636-9cbc-48617b739f84)
![Screenshot 2024-11-20 220603](https://github.com/user-attachments/assets/fcbeb9fd-9eea-4c49-915f-32589a64478a)

**Reply by jonwaldstein** [reply]
That's weird, i'll have to reach back out to Stripe and ask why this is happening.

**Reply by demo7up** [reply]
![Screenshot 2024-12-10 203852](https://github.com/user-attachments/assets/efecbe3d-2b2b-4503-a4cf-c5cc21a03de3)

Seems like its half fixed the first part has now updated and changed but the Data Encryption portion has retained the GiveWP title/name. 

Its definitely progress!

**Reply by jonwaldstein** [reply]
Oh great! Thanks for the update, I wonder if it just took some time to populate 🤔 I suppose we still need to check with Stripe about that data encrypted part
